resources/views/models/view.blade.php

```diff
@@ -344,7 +344,10 @@ class="table table-striped snipe-table"
                 @else
 
                     <div class="col-md-12" style="padding-bottom: 10px;">
-                        <a href="{{ route('models.destroy', $model->id) }}" style="width: 100%;" class="btn btn-sm btn-danger hidden-print">{{ trans('general.delete') }}</a>
+                        <form action="{{ route('models.destroy', $model->id)  }}" method='POST'>
```

Comment:
Wouldn't this bypass the warning modal tho?

Reply:
IIRC, the form should be in the modal

Reply:
Well, I thought: the user is already in the desired Asset model, and the Delete button is big and red... No room to mistakes I thought...
I also thought: I don't know how to bring up the Modal 😆 

But sure, let me keep looking how we do it in the list view before this so I can replicate it.